apps/test-suite/tests/ImageManipulator.js

```diff
@@ -24,6 +24,18 @@ export async function test(t) {
         t.expect(typeof result.height).toBe('number');
       });
 
+      t.it('returns valid image from base64 data URL', async () => {
+        // 1x1 red image
+        const result = await ImageManipulator.manipulateAsync(
+          'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8DwHwAFBQIAX8jx0gAAAABJRU5ErkJggg==',
+          [{ resize: { width: 100, height: 100 } }]
+        );
+        t.expect(result).toBeDefined();
+        t.expect(typeof result.uri).toBe('string');
+        t.expect(typeof result.width).toBe('number');
+        t.expect(typeof result.height).toBe('number');
```

Comment:
Can't we be more specific about the results here? 🤔 
```suggestion
        t.expect(result.width).toBe(100);
        t.expect(result.height).toBe(100);
```

Reply:
I copied what the test above it does, I've changed it but I think the intention of both was more can it read and process the file (from local file or base64) not whether the resize action is doing what is requested.